navidrome/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: navidrome_server_1
+      APP_PORT: 4533
+      PROXY_AUTH_ADD: "false"
+  server:
+    image: deluan/navidrome:0.52.5@sha256:b154aebe8b33bae82c500ad0a3eb743e31da54c3bfb4e7cc3054b9a919b685c7
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
+      - ${UMBREL_ROOT}/data/storage/music:/music:rw
+
+# You do not need to expose the port that your app's web server is listening on if you're using the app_proxy service.
+      # This is handled by the APP_HOST and APP_PORT environment variables in the service above.
+      #
+      # If you need to expose additional ports, you can do so like this, replacing <port> with the port number:
```

Comment:
You accidentially left some comments in here
```suggestion
```

Reply:
Or did you add them on purpose?

Reply:
It really was an accident, I only realized it with your warning.